src/Makefile

```diff
@@ -12,11 +12,19 @@ AS=arm-none-eabi-as
 # Specity the linker to use
 LD=arm-none-eabi-ld
 
-CCFLAGS=-mcpu=cortex-m0 -mthumb -g -ffreestanding -I ../include
+BOARD=STM32F042
 
+CCFLAGS=-mcpu=cortex-m0 -mthumb -g -D$(BOARD) -I ../include
+CCFLAGS+= -I libs/Libraries/CMSIS/Device/ST/STM32F0xx/Include
+CCFLAGS+= -I libs/Libraries/CMSIS/Include
+CCFLAGS+= -I libs/Libraries/STM32F0xx_StdPeriph_Driver/inc
+
+#$(wildcard libs/Libraries/STM32F0xx_StdPeriph_Driver/src/*.c)
 # List the object files involved in this project
-LIBS=stdio.o uart.o
-OBJS=init.o main.o $(LIBS)
+SRCS=$(wildcard *.c) \
+		 libs/Libraries/STM32F0xx_StdPeriph_Driver/src/stm32f0xx_rtc.c \
+		 libs/Libraries/STM32F0xx_StdPeriph_Driver/src/stm32f0xx_rcc.c
```

Comment:
@MaxMSchweitzer Good question! We must only include what we need, This expands `.text` a bit because ST provides three functions for setting up the clock, the rest is all just driver code which we can toggle including or not here